README.md

```diff
@@ -3,17 +3,17 @@
 
 ## Table of Contents
 - [Introduction](#introduction)
-- [Getting Started](#getting-started)
-- [Contributing](#contributing)
+- [End Users](#end-users)
+- [Developers](#developers)
 
 ## Introduction
 BonnyCI is a Continuous Integration service offered to developers of Open Source projects.
 
-## Getting Started
-Checkout the [getting started](getting_started) section.
+## End Users
+Check out the [end user](end_users) documentation.
 
-## Contributing
-Please see the [contribution documentation](contributing) for more information.
+## Developers
+Please see the [developer documentation](developers) for more information.
```

Comment:
Same comment as above (making the link text "developers" would make this sentence read poorly). Additionally, including "documentation" in the link text was what I meant to do in both the developer and end user sections.